rskj-core/src/test/java/co/rsk/peg/utils/BridgeEventLoggerImplTest.java

```diff
@@ -566,6 +566,18 @@ public void testLogReleaseBtcRequestRejected() {
         assertEvent(eventLogs, 0, BridgeEvents.RELEASE_REQUEST_REJECTED.getEvent(), new Object[]{sender}, new Object[]{amount.value, reason.getValue()});
     }
 
+    @Test
+    public void logBatchPegoutCreated() {
```

Comment:
Test with a list of rsk tx hashes, not just one

Reply:
Done